src/main/resources/org/jenkinsci/plugins/gwt/global/JobFinderConfigPathItem/help-path.html

```diff
@@ -0,0 +1,7 @@
+For example:
+<ul>
+    <li><b>jobName:</b> The jobName will be triggered every time</li>
+    <li><b>/path/+pull_request.base.repo.namespace+/+jobName:</b> The path of jobName specified in the request body will be triggered</li>
```

Comment:
I think it is better to keep the variable syntax used in the jobs. So this would be `${pull_request.base.repo.namespace}`.

Reply:
Yes, You are right, There have a bug if use '+' to concat, Because the directory and job name can include in '+',  cant include "$", I will fix it.